### ISSUE TITLE
[Feature Request] Closed captions remember state

### ISSUE THREAD

**Issue by strapped-puppy** [futo-org/grayjay-android#451]
I'd like the captions to remember if you turn them on (when available) so a user doesn't need to manually turn them on for each video.

This would significantly improve accessibility in the app.

**Comment by McIrco95** [comment]
There should be at least a setting to turn cc on as default. 

Would be great if that setting could be found via search (cc, captions, subtitles)

It would be great if activating the cc multiple times in a row would ask if you want to change the default. (Same when disabling multiple times) 

could also be implemented for playback speed, video quality and language. So that if you do something over and over the app asks you to make it the default

**Comment by KraXen72** [comment]
this is a very important feature! it's extremly cumbersome to have to re-enable subtitles and change playback speed after every video.

**Comment by gilgulgamesh** [comment]
Yeah this is definitely an accessibility issue. It's better low-priority or QOL. The fact that subtitles are uncustomisable from within the app is rough, but at least they respects the built in android captioning settings. **What isn't respected is the system setting to have CC on by default.** There isn't even a quick way to enable them.

Newpipe's implementation of sending the user to configure CC in android settings works great and seems simple. All that's missing is one button in settings and to respecting one toggle!

**Comment by defiling9046** [comment]
@Zvonimir-FUTO This can be marked as completed since it now remembers whether you had captions on or off from the last video you watched and then remembers that for the next video you watch.
